mysql-test/suite/federated/federated_bug_lp1739734.test

```diff
@@ -0,0 +1,53 @@
+--echo #
```

Comment:
Please check the Confluence page on the testcase naming, it should be named percona_bug1739734